cmd/admin-update.go

```diff
@@ -106,7 +113,9 @@ func mainAdminServerUpdate(ctx *cli.Context) error {
 
 	updateURL := args.Get(1)
 
-	if isTerminal() {
+	autoConfirm := ctx.Bool("yes")
+
+	if isTerminal() && !autoConfirm {
```

Comment:
No issue with this but I just wanted to understand how isTerminal() is not enough, in which environment is this needed ?

Reply:
Not sure how the prompt could be bypassed due to `isTerminal()` by adding the explicit flag and showing it on help users can discover this